R/showMultiLinePlotsByVariable.R

```diff
@@ -92,39 +97,46 @@ showMultiLinePlotsByVariable <- function(
 
   label <- paste0("(", paste0(levels(d$unit), collapse = ","), ")")
   xLabel <- paste0(xVar, " (", paste0(levels(d$unit.x), collapse = ","), ")")
-
+  
+  if (showGlobal) {
   p1 <- dMainScen %>%
```

Comment:
Please double check the indentation here, I think this needs to be indented more because you added the `if`